bin/includes/preamble.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$0")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+
+if [ -n "$1" ]; then
```

Comment:
I'm still getting an error message when I run `bin/cloudgov/deploy-waf`: 

```
usagov-2021/bin/includes/preamble.sh: line 16: $1: unbound variable
```

I suggest doing something like this: 

```
FIRST=${1:-}
if [ -n "$FIRST" ]; then
```